apps/dashboard/app/views/support_ticket/email_service_template.html.erb

```diff
@@ -0,0 +1,92 @@
+<%= javascript_include_tag 'support_ticket', nonce: true %>
+
+<script>
+
+    const SUPPORT_TICKET_RESTRICTIONS = <%= raw AttachmentsValidator.restrictions.to_json %>
+    const SUPPORT_TICKET_MESSAGES = {
+        "items.attachments": "<%= t('dashboard.support_ticket.validation.items.attachments_js') %>",
+        "size.attachments": "<%= t('dashboard.support_ticket.validation.size.attachments_js') %>",
+    }
+
+</script>
```

Comment:
Can you wrap this in a `<%= javascript_tag nonce: true do -%>` instead of hard coding the `<script>` so it can pick up a nonce.

Reply:
Updating...